wrappers/services_test.go

```diff
@@ -435,7 +451,9 @@ func (s *servicesTestSuite) TestEnsureSnapServicesWithZeroCpuCountAndCpuSetQuota
 	c.Assert(err, IsNil)
 
 	m := map[*snap.Info]*wrappers.SnapServiceOptions{
-		info: {QuotaGroup: grp},
```

Comment:
(niptick) this looks like whitespace changes?

Reply:
Undid this and many others in services_test.go!